warp/src/main/java/com/schibsted/nmp/warp/theme/WarpIcons.kt

```diff
@@ -0,0 +1,2116 @@
+package com.schibsted.nmp.warp.theme
+
+import android.annotation.SuppressLint
+import android.content.Context
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import com.schibsted.nmp.warp.R
+
+/**
+ * A collection of Warp icons. Default size 32x32dp.
+ */
+open class WarpIconResource(
+    open val vector: ImageVector,
+    open val description: String
+)
+
+/**
+ * A collection of Warp taxonomy icons.
+ * Can be retrieved by unique identifier which is identical across all platforms.
+ */
+
+open class WarpTaxonomyIconResource(
+    override val vector: ImageVector,
+    override val description: String,
+    val identifier: String
+) : WarpIconResource(
+    vector = vector,
+    description = description
+)
+
+class WarpBrandIconResource(
+    override val vector: ImageVector,
+    override val description: String,
+    val identifier: String
+) : WarpIconResource(
+    vector = vector,
+    description = description
+)
+
+object WarpIconResources {
+
```

Comment:
I think it would still be useful to get a helper method that would just allow the user to check for the existence of an icon from its identifier.
It should check _**all**_ types of icons (so `taxonomy` but also `WarpBrandIconResource`)
It should simply return a Boolean.
of course prefereably without the need for anything more than the identifier string, but I would settle for having to pass a `Context` also if needed